ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/conversion_filters/112393_filter-by-region.cy.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types ='Cypress'/>
+
+Cypress._.each(['ipad-mini'], (viewport) => {
+    describe(`112393: Filter projects by region ${viewport}`, () => {
+      beforeEach(() => {
+        cy.viewport(viewport);
+        cy.login();
+        cy.navigateToFilterProjects();
+      })
+  
+      afterEach(() => {
+        cy.clearCookies();
+      });
+  
+      it('TC01: should display results when region option is checked', () => {
```

Comment:
Is it worth adding a test, which returns a few results?

Reply:
To verify if Region filter is working or not this needs to be included. It might return more results based on Data